STEERING-COMMITTEE.md

```diff
@@ -162,13 +117,157 @@ There are two ways to raise issues to the steering committee for decision:
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
 
+## Composition
+
+The steering committee will have 7 seats.  Five of the seats will be
```

Comment:
It'd be worth working out the use of tenses. "Will have" future tense or "has" present tense. I'd suggest present tense, it's always easier to read.

Reply:
Agree, and I'm changing it wherever it makes sense (the language about elections seems appropriate to use future tense in since it is describing the next election, which hasn't happened yet at any given time)